migrations/20170428154714_multiple_api_tokens/up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE api_tokens (
+  id SERIAL PRIMARY KEY,
+  user_id integer NOT NULL REFERENCES users(id),
+  token character varying DEFAULT random_string(32) NOT NULL UNIQUE,
```

Comment:
This is super niche, but can we use `text` instead of `varchar` here? They're identical under the hood, but are considered different types and specifically have conversion issues if you were to use `array_agg` or similar.